### ISSUE TITLE
Rename WIP status to Draft

### ISSUE THREAD

**Issue by bmeneg** [xanzy/go-gitlab#1168]
GitLab is planning to fully remove the `WIP` word from their `Draft` status [1] in their 14.0 version, which is about to be released.
It still not clear what's going to happen in the API side, but it would be nice to rename the `WIP` struct member from `ListMergeRequestsOptions` to `Draft` to properly represent the terminology.

```
Adding WIP: to the start of the merge request’s title still marks a merge request as a draft.
This feature is scheduled for removal in GitLab 14.0. Use Draft: instead.
```
Source: [2]

[1] https://gitlab.com/gitlab-org/gitlab/-/issues/228685
[2] https://docs.gitlab.com/ee/user/project/merge_requests/drafts.html

**Comment by bvicinus** [comment]
documentation on how WIP is being deprecated in favor of Draft, along with a slew of other changes. 

https://about.gitlab.com/blog/2021/06/04/gitlab-moving-to-14-breaking-changes/

**Comment by Deichindianer** [comment]
Did create the field in my PR.
Tests run but I think there is more discussion to be had around what this really means, the docs are a bit confusing as well as the behaviour of GitLab itself on this one :thinking:

**Comment by svanharmelen** [comment]
"fixed" by #1283
